moderation/NicknameModeration.go.tmpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+	Custom Command for nickname moderation
+	If a user of your server has an inappropriate or unpingable nickname, mods often have to change their nickname manually.
+	With this custom command, they can moderate nicknames with a simple command.
+	After you moderated the nickname, the nickname will be changed when the moderated user sends their next message.
+	The command nick-numberreset makes the next moderated nickname start at number one.
+
+	Trigger type: Regex
+	Trigger: \A
+
+	Usage:
+	-modnick <@user/ID>
+	-unmodnick <@user/ID>
+	-nick-numberreset
+
+	Credit: 2x2Master#7363
+	Feel free to ping me in YAGPDB support if you have any questions or problems!
+*/}}
+
+{{/* Configuration area starts */}}
+{{$prefix := "-"}}{{/* YAGPDB's prefix in your server */}}
```

Comment:
Would remove this line and add it further down, filling the variable with the following:
```go
{{$prefix := index (reFindAllSubmatches `.*?: \x60(.*)\x60\z` (execAdmin "prefix")) 0 1}}
```` 
As David said, I'm not a fan of the prefix as a user variable either...